client/src/components/PhotosModal.jsx

```diff
@@ -17,17 +18,27 @@ const PhotosModalWrapper = styled.div`
   };
 `;
 
+const PhotosModalEl = styled.div`
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+`;
+
+const PrimaryPhotoWrapper = styled.div`
+  margin-bottom: 2.5em;
+  height: auto;
+`;
+
 const PrimaryPhoto = styled.img`
-  margin: 0;
-  position: absolute;
-  top: 50%;
-  transform: translateY(-50%);
   max-width: 100%;
   height: auto;
-  padding-left: 10%;
-  &:hover {
-    cursor: pointer;
-  }
+`;
+
+const ProductInfoWrapper = styled.div`
```

Comment:
Can you take out these empty styled components?

Reply:
I left them there just in case I need to add some styling to the divs in the future. I could also replace them with regular divs.